server/devpi_server/model.py

```diff
@@ -134,10 +134,14 @@ def getstage(self, user, index=None):
 
     def is_empty(self):
```

Comment:
Might be useful to rename this method to `may_import`.
It seems to be used only by `./server/devpi_server/importexport.py`